pkg/crc/machine/machine.go

```diff
@@ -392,14 +398,13 @@ func Stop(stopConfig StopConfig) (StopResult, error) {
 		return *result, errors.New(err.Error())
 	}
 
-	libMachineAPIClient := libmachine.NewClient(constants.MachineBaseDir, constants.MachineCertsDir)
-	host, err := libMachineAPIClient.Load(stopConfig.Name)
-
+	libMachineAPIClient, host, err := GetHost(stopConfig.Name)
 	if err != nil {
 		result.Success = false
 		result.Error = err.Error()
 		return *result, errors.New(err.Error())
 	}
+	defer libMachineAPIClient.Close()
```

Comment:
did you test each command with the addition of this libMachineAPIClient.Close()? I think I tried to add this some time ago, and that in some commands it was causing a crash.

Reply:
I didn't check for all the commands but checked for delete/stop and that's working, I will check for others and see if that cause any issue.

Reply:
@cfergeau checked all the commands, didn't see any issue about causing a crash.